IOScanco wrapping fails do to fatal error: 'itkCommand.h' file not found

String `itkCommand.h` is not found in IOScanco's sources. So it must be coming from ITK module/wrapping infrastructure. This is most likely caused by recent refactoring to the build system and wrapping infrastructure.

### Expected behavior

Successful build.

### Actual behavior

```log
1>------ Build started: Project: IOScancoCastXML, Configuration: Release x64 ------
1>Generating ../../castxml_inputs/itkScancoImageIO.xml
1>C:/Dev/ITK-vs22/Wrapping/castxml_inputs/itkScancoImageIO.cxx:1:10: fatal error: 'itkCommand.h' file not found
1>    1 | #include "itkCommand.h"
1>      |          ^~~~~~~~~~~~~~
1>1 error generated.
1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'C:\Dev\ITK-vs22\CMakeFiles\49fd428f9683d364515c70d2494df883\itkScancoImageIO.xml.rule;C:\Dev\ITK-vs22\CMakeFiles\3a5d818ec641f8a39ddcc8e824cdd57f\IOScancoCastXML.rule' exited with code 1.
1>Done building project "IOScancoCastXML.vcxproj" -- FAILED.
```

### Reproducibility

Happened on a somewhat clean build on Windows, and reproduced on a completely clean build on Ubuntu 22.04.

### Versions

ITK version: cb4aa250351fe11429c22f83e1e4635fc5b5a8ef and d26e43655faa8fe6add9c1da6da6cf48b57cf462.
Module_IOScanco_GIT_TAG: ab86ac60da63245a41daca897f19fe02f38b1952 (latest)

### Environment

Windows, latest CMake (4.2.3) and VS2022 (Version 17.14.27 from February 2026).

--- a/Modules/Remote/IOScanco.remote.cmake
+++ b/Modules/Remote/IOScanco.remote.cmake
@@ -46,5 +46,5 @@ itk_fetch_module(
   "An ITK module to read and write Scanco microCT .isq files."
   MODULE_COMPLIANCE_LEVEL 2
   GIT_REPOSITORY https://github.com/InsightSoftwareConsortium/ITKIOScanco.git
-  GIT_TAG ab86ac60da63245a41daca897f19fe02f38b1952
+  GIT_TAG 2209183eac3d4c62de8fb5c8f42e4ac86e634c83
   )

